Update server.regional.cfm

andreasRu opened the pull request:
Add information that  "server date/time" is the JVM time and not related to the server-context timezone setting. 
See this thread: https://dev.lucee.org/t/server-admin-is-showing-a-contexts-date-time-why/11775/7

andreasRu committed:
Update server.regional.cfm

Add information that  "server date/time" is the JVM time and not related to the server-context timezone setting. 
See this thread: https://dev.lucee.org/t/server-admin-is-showing-a-contexts-date-time-why/11775/7

andreasRu commented:
Please excuse that I didn't create an LDEV ticket. That is such a minor change that I didn't want to bother anybody with this. If you want you can of course reject and silently close this.

zspitzer commented:
nice tweak!

no need for a ticket, but can you change that in the lang files instead, ie en.xml etc

andreasRu committed:
Merge branch 'lucee:6.0' into patch-3

andreasRu committed:
Update server.regional.cfm

Remove "jvm" string from page for adding it to the language xml files

andreasRu committed:
Update en.xml

Add "JVM" to overview.servertime

andreasRu committed:
add the string JVM to servertime

Please see https://dev.lucee.org/t/server-admin-is-showing-a-contexts-date-time-why/11775/8

andreasRu committed:
Add JVM to overview.servertime

Please see https://dev.lucee.org/t/server-admin-is-showing-a-contexts-date-time-why/11775/8

andreasRu committed:
Add JVM to overview.servertime

Please see https://dev.lucee.org/t/server-admin-is-showing-a-contexts-date-time-why/11775/8

andreasRu commented:
@zspitzer I've corrected and added it to all available language files.
Merry Christmas to you all!
